src/components/Typography/Typography.module.css

```diff
@@ -18,158 +18,135 @@ limitations under the License.
  * Font – Body – Extra Small
  */
 
-[class^="font-body-xs"] {
```

Comment:
Looking at the CSS output file makes it obvious that this trick does not work with CSS modules